### ISSUE TITLE
Great job !

### ISSUE THREAD

**Issue by paulgirard** [tategallery/collection#14]
You rock !

A great dataset, great format, great... 

Thanks for sharing

**Comment by liamca** [comment]
Yes, this dataset is amazing!  I created a demo to show how to do full text searching of the dataset here: https://github.com/liamca/azure-search-tate-art-gallery
